deploy/terraform/digitalocean/README.md

```diff
@@ -0,0 +1,21 @@
+# Digital Ocean
+
+This terraform module sets up and configures a k8s cluster as a deployment target for both Datadog and the Ecommerce application.
+
+## Initial Setup
+
+* Export the following environment variable:
+    * `TF_VAR_do_token` with your DigitalOcean API Token
+* Review the variables in the `variables.tf` file if you want to make any adjustments
+* Run `terraform init` to install all of the needed terraform modules
+* Run `terraform apply` to spin up the cluster. The location of the cluster will output at the end.
+
+## Automatic `kubectl` config
+
+When you `terraform apply` and it is successful, this terraform configuration will automatically write a `kube_config_server.yaml` file to use with `kubectl`. To automatically use that config, you need to export the `KUBECONFIG` environment variable pointing to this file like so:
+
+```bash
+export KUBECONFIG="$(pwd)/kube_config_server.yaml"
+```
+
+If you use [direnv](https://direnv.net/) you can put the above line into an `.envrc` in this directory to automatically load the config for you each time you visit this directory.
```

Comment:
If in the end we are not deploying the ecommerce app (yet) with Terraform, shall we point from here to the deploy/kubernetes folder so people know where to continue?

Reply:
That would be good. I will point to the generic-k8s folder for now then and update when I have a HELM chart ready.